Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/InputActionsEditorWindowUtils.cs

```diff
@@ -14,12 +14,15 @@ internal class InputActionsEditorWindowUtils
         public static void SaveAsset(SerializedObject serializedAsset)
         {
             var asset = (InputActionAsset)serializedAsset.targetObject;
+
             // for the global actions asset: save differently (as it is a yaml file and not a json)
             if (asset.name == ProjectWideActionsAsset.kAssetName)
             {
                 AssetDatabase.SaveAssets();
+                ProjectWideActionsAsset.CreateRoslynAdditionalFile();
```

Comment:
If we tracked the path to .additionalfile via asset or via InputActionAsset serialised property I assume we could have e.g. 

```
if (asset.m_RoslynCodeGenerationAssetPath != null)
     ProjectWideActionsAsset.CreateRoslynAdditionalFile();
```
     
     or similarly via another separate serialised object stored within InputManager.asset we could determine the same thing and/or also use it as a setting?